server/test/read.go

```diff
@@ -673,11 +567,119 @@ func ReadQueryErrorTest(t *testing.T, datastore storage.OpenFGADatastore) {
 			require.NoError(err)
 
 			test.request.StoreId = store
-			if test.request.AuthorizationModelId == "" {
-				test.request.AuthorizationModelId = test.model.Id
-			}
 			_, err = commands.NewReadQuery(datastore, tracer, logger, encoder).Execute(ctx, test.request)
-			require.EqualError(err, test.err.Error())
+			require.Error(err)
 		})
 	}
 }
+
+func ReadAllTuplesTest(t *testing.T, datastore storage.OpenFGADatastore) {
+	ctx := context.Background()
+	tracer := telemetry.NewNoopTracer()
+	logger := logger.NewNoopLogger()
+	store := ulid.Make().String()
+
+	model := &openfgapb.AuthorizationModel{
+		Id:            ulid.Make().String(),
+		SchemaVersion: typesystem.SchemaVersion1_0,
+		TypeDefinitions: []*openfgapb.TypeDefinition{
+			{
+				Type: "repo",
+			},
+		},
+	}
+
+	err := datastore.WriteAuthorizationModel(ctx, store, model)
```

Comment:
Why is this write necessary if the `Read` API no longer requires an AuthorizationModel to exist before serving the Read query? Shouldn't it suffice to just write the tuples and then call `Read` without a `tuple_key` field?

Reply:
Good point. I thought I needed it for write, but that is only at the server/command layer. Done. Thanks!